nginx/ngx_http_js_module.c

```diff
@@ -1245,7 +1251,8 @@ static ngx_int_t
 ngx_http_js_variable_set(ngx_http_request_t *r, ngx_http_variable_value_t *v,
     uintptr_t data)
 {
-    ngx_str_t *fname = (ngx_str_t *) data;
+    ngx_http_js_set_t *vdata = (ngx_http_js_set_t *) data;
```

Comment:
Style: sort lines in type length order, ie

```
ngx_str_t *fname = &vdata->fname;
ngx_js_set_t *vdata = (ngx_js_set_t *) data;
```

Reply:
`fname` needs `vdata` defined (and casted). I've moved it to variables declarations and set it below instead. Hopefully it looks better now